cv19ResSupportV3.Tests/V3/Factories/Commands/CreateHelpRequestCallFactoryTest.cs

```diff
@@ -0,0 +1,35 @@
+using AutoFixture;
+using cv19ResSupportV3.V3.Boundary.Requests;
+using cv19ResSupportV3.V3.Domain.Commands;
+using cv19ResSupportV3.V3.Factories.Commands;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace cv19ResSupportV3.Tests.V3.Factories.Commands
+{
+    [TestFixture]
+    public class CreateHelpRequestCallFactoryTest
+    {
+
+        [Test]
+        public void RequestToCommand()
+        {
+            var request = new Fixture().Build<CreateHelpRequestCallRequest>().Create();
+
+            var command = request.ToCommand();
+
+            command.HelpRequestId.Should().Be(request.HelpRequestId);
+        }
+
+        [Test]
+        public void CommandToEntity()
+        {
+            var command = new Fixture().Build<CreateHelpRequestCall>().Create();
```

Comment:
Might be outside the scope of this PR, but since I'm not in the next week I'll raise this up anyway:
It's better practice to factor out the `new Fixture()` into a separate static class accessible for all the test files rather than instantiating it for every test. Example: [Randomm](https://github.com/LBHackney-IT/lbh-fss-public-api/blob/74e63a5d4ef8c807d1858ef4f9e89b40d1e3b8c2/LBHFSSPublicAPI.Tests/TestHelpers/Randomm.cs), [EntityHelper](https://github.com/LBHackney-IT/lbh-fss-public-api/blob/74e63a5d4ef8c807d1858ef4f9e89b40d1e3b8c2/LBHFSSPublicAPI.Tests/TestHelpers/EntityHelpers.cs). Or at the very least you can extract the fixture to the top of this test class as a class field, which could be accessed by every test within this test class.

Reply:
Hmm... my thinking was that extracting this randomising functionality away is about making the tests' setup look cleaner and easier to modify once bigger changes occur _(as you don't have to go around changing every instance that uses the same pattern)_. Since this is only used for tests, I figured a single Tests helper class would suffice.